src/components/Post/post.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Inputs from '../../components/Inputs/inputs';
+
+import axios from 'axios';
+
+class Post extends Component {
+
+  constructor(){
+    super();
+    this.state = {
+      name: ""
+    }
+  }
+
+
+  handleInputData = (name, value) => {
+    this.setState({[name]:value});
+  }
+
+
+  fileUploadHandler = () => {
+
+    const formData = new FormData();
+    const config = {headers: { 'content-type': 'multipart/form-data' }}
+
+    formData.append('owner', this.state.owner);
+    formData.append('title', [this.state.month, this.state.year]);
+    formData.append('uploadFile', this.state.selectedFile);
+
+    axios.post('http://localhost:5000/api/upload-data', formData, config)
+      .then( res => console.log(res, this.state))
```

Comment:
I think `axios` has some config properties to specify the `base host` of the API Calls. :)

Something like [Creating an instance](https://github.com/axios/axios#creating-an-instance)

Reply:
I will look into that

Reply:
IMO creating a service file to abstract the whole API appiying @jalmarazg changes should be the best approach.

Reply:
it's Ready!